ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -237,9 +305,13 @@ main() {
   if [[ $# -lt 1 ]]; then
     usage
   fi
+  if [[ -z "$GPHOME" ]]; then
+    GPHOME="/usr/local/hawq"
```

Comment:
we do not need GPHOME, just use relative path to current script. No reason to ask users to source extra files.